### ISSUE TITLE
Remove dependency on Zstd (from ZipWriter) for easier wasm setup

### ISSUE THREAD

**Issue by rRaDuCaN** [jmcnamara/rust_xlsxwriter#11]
Provide a lighter version of the lib without any c lib dependencies, for a possibly, web assembly integration. Zstd is driving me crazy with wasm setup.

**Comment by jmcnamara** [comment]
I've pushed a change to main to reduce the zipwriter dependencies down to just "defalate" and effectively removing the "zstd" dependency. Can you try that and let me know how you get on.

**Comment by jmcnamara** [comment]
And just in case it isn't clear you can test it by changing your dependency to this:

```toml
[dependencies]
rust_xlsxwriter = { git = "https://github.com/jmcnamara/rust_xlsxwriter" }
```

**Comment by jmcnamara** [comment]
See also the changes in #12 which might help.

**Comment by rRaDuCaN** [comment]
@jmcnamara , works like a charm, thank you so much.

**Comment by jmcnamara** [comment]
@rRaDuCaN That's great. Thanks for letting me know. I'll push out the change shortly to crates.io in a dot release.

Btw, would you care to share the step you did to did to get it working with wasm. I plan to add a section about that in the User Guide in the next few week so any workaround or things to look out for, for the end user, would be helpful.

**Comment by jmcnamara** [comment]
Thanks for the suggestion.

Fixed in version 0.12.1. Closing.
